### ISSUE TITLE
Can't get Lashi to load

### ISSUE THREAD

**Issue by nh36** [knightss27/capr#11]
The Lashi transducer is now in place and there are no error messages, but I can't get any Lashi data to load. I wonder if somehow in the refishing process in the back end it isn't looking for Lashi.

**Comment by knightss27** [comment]
And you have the `save stack lashi.bin` at the end of that part? I wonder if there might be more notation needed, since some of the other ones have `push` with certain functions before the `save stack`.

**Comment by nh36** [comment]
[capr-fst-8_6_2022.txt](https://github.com/knightss27/capr/files/8864925/capr-fst-8_6_2022.txt)

Here's my current version for your delectation. Seems to have push and save stack in the usual places. shrug...

**Comment by knightss27** [comment]
Yes at first glance that does look like it should work. I will investigate what the server is expecting if you add a transducer.
